lib/doorkeeper/config.rb

```diff
@@ -623,6 +632,11 @@ def token_grant_types
     def deprecated_token_grant_types_resolver
       @deprecated_token_grant_types ||= calculate_token_grant_types
     end
+    
+    def native_authorization_code_route
+      @opt_out_native_route_change ||= false
```

Comment:
Let's use `unless defined?(@opt_out_native_route_change)` to be safe

Reply:
makes sense, updated!